Extensions/Video/JsExtension.js

```diff
@@ -0,0 +1,775 @@
+
+    object
+      .addExpression(
+        "IsLooped",
+        _("Video is looped"),
+        _("Return if video is looped"),
+        _("Time"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isLooped");
+
+    object
+      .addCondition(
+        "Volume",
+        _("Volume"),
+        _("Test the volume of an video object"),
+        _("Volume of _PARAM0_ is _PARAM1_ _PARAM2_"),
+        "Volume",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .addParameter("relationalOperator", _("Sign of the test"))
+      .addParameter("expression", _("Value 0-100"))
+      .getCodeExtraInformation()
+      .setFunctionName("getVolume")
+      .setManipulatedType("number");
+
+    object
+      .addCondition(
+        "Mute",
+        _("Is muted"),
+        _("Test if an video is muted"),
+        _("_PARAM0_  is muted"),
+        "Volume",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isMuted");
+
+    object
+      .addExpression(
+        "GetCurrentTime",
+        _("Get current time"),
+        _("Return the current time of an video object (in seconds)"),
+        _("Time"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("getCurrentTime");
+
+    object
+      .addExpression(
+        "GetDuration",
+        _("Get the duration"),
+        _("Return the duration of an video object (in seconds)"),
+        _("Time"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("getDuration");
+
+    object
+      .addExpression(
+        "IsEnded",
+        _("Get the duration"),
+        _("Get the duration of an video object (in seconds)"),
+        _("Time"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isEnded");
+
+    object
+      .addCondition(
+        "Ended",
+        _("Is ended"),
+        _("Is ended"),
+        _("_PARAM0_ is ended"),
+        _("Control time"),
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isEnded");
+
+    object
+      .addAction(
+        "SetOpacity",
+        _("Set opacity (in %)"),
+        _("Set opacity (in %)"),
+        _("Do _PARAM1_ _PARAM2_ % to the opacity of _PARAM0_"),
```

Comment:
As before, no extra space:
`Do _PARAM1__PARAM2_ to the opacity of _PARAM0_`

I let you check other actions.